helm/app-operator/templates/_resource.tpl

```diff
@@ -26,3 +26,7 @@ room for such suffix.
 {{- define "resource.default.namespace" -}}
 giantswarm
 {{- end -}}
+
+{{- define "resource.app.unique" -}}
+{{ if eq .Release.Name "app-operator-unique" }}true{{ else }}false{{ end }}
+{{- end -}}
```

Comment:
This is the magic that decides if the operator is running as unique or not.

Reply:
I’d strongly prefer to use `hasSuffix "-unique"` here as this file is fully copy-pasteable without any modifications. Otherwise LGTM.

Reply:
Thx! Done.